cmake/BuildICUB.cmake

```diff
@@ -81,3 +82,16 @@ ycm_ep_helper(ICUB TYPE GIT
                                     -DENABLE_icubmod_socketcan:BOOL=${ENABLE_icubmod_socketcan}
                                     -DICUB_USE_icub_firmware_shared:BOOL=${ROBOTOLOGY_ENABLE_ICUB_HEAD}
                                     -DICUBMAIN_COMPILE_SIMULATORS:BOOL=${ICUBMAIN_COMPILE_SIMULATORS})
+
+# Options related to generation of conda binary packages
+set(ICUB_CONDA_DEPENDENCIES ace opencv gsl ipopt libode qt sdl)
+if(NOT (APPLE OR WIN32))
+  list(APPEND ICUB_CONDA_DEPENDENCIES libdc1394)
+endif()
+
+if(NOT APPLE)
+  list(APPEND ICUB_CONDA_DEPENDENCIES freeglut)
+endif()
+if(CMAKE_SYSTEM_NAME STREQUAL "Linux")
```

Comment:
I've often seen "UNIX AND NOT APPLE" - not sure which one is better.

Reply:
`UNIX AND NOT APPLE` is typically used to include *bsd systems, as conda-forge do not support any *bsd or similar target at the moment I think it is more clear to indicate it as "do something if we are on linux". If conda-forge will ever support other non-Linux targets (such as WebAssembly ( https://github.com/iodide-project/pyodide/issues/795 ), that would be a dream!) we will eventually fix this.